netfile/management/commands/downloadnetfilerawdata.py

```diff
@@ -263,7 +273,20 @@ def fetch_transactions_agency_year(self, agency, year):
 
     def fetch_agencies(self):
         """Fetches agencies from Netfile API"""
-        agencies = self.connect2.getpubliccampaignagencies()
-        print "Found %s agencies" % (len(agencies))
-        self._write_csv('netfile_agency.csv', iter(agencies))
+        agency_csv_path = op.join(self.data_dir, 'netfile_agency.csv')
+        agencies = None
+        if not self.force and op.exists(agency_csv_path):
+            import pandas as pd
+            try:
+                agencies = pd.read_csv(agency_csv_path)
+                agencies = agencies.T.to_dict().values()
```

Comment:
I'm using `pandas` to do all of the data filtering in https://github.com/caciviclab/disclosure-backend/pull/124 . It's also kind of a lynchpin of data cleaning and data science.

Is it that severe of a dependency? It would take significant work to extract it there, and would definitely slow down my progress on trying to filter the data and shove it in the DB.